tests/ui/consts/const-eval/transmute-const.rs

```diff
@@ -1,4 +1,3 @@
-//@ stderr-per-bitwidth
```

Comment:
This is unrelated to the PR, I just noticed there's no reason to have per-bitwidth error files here.